parm/fv3lam_rrfs.xml

```diff
@@ -575,6 +612,12 @@
        <scale>3.0</scale>
        </param>
 
+    <param>
+       <shortname>GFS_4LFTX_ON_SURFACE</shortname>
```

Comment:
@AndrewBenjamin-NOAA May I know if this product is available in which regional applications, e.g. rap, hrrr, nam?

Reply:
This is from the fv3 and arw hiresw. Those models produce 4LFTX on the surface every 3hrs from 00-60 hours.

Reply:
@AndrewBenjamin-NOAA Got it.

Reply:
@AndrewBenjamin-NOAA If it is only every 3 h, it is almost certainly smartinit output.  Are you seeing it in the 2p5km output?

Reply:
@AndrewBenjamin-NOAA @MatthewPyle-NOAA I am confused. Has this product in ops. hiresw been generated from UPP or other downstream utilities, e.g. smartinit?